ansible_collections/arista/avd/molecule/cv_deploy/intended/structured_configs/base_configs/avd-ci-leaf2.yml

```diff
@@ -0,0 +1,2 @@
+hostname: avd-ci-leaf2
```

Comment:
is_deployed is default true, so it should not be needed to have it in every file.